<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>proe-19l-2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Elements</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>Designator</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Type</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>X</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Y</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Line Width</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Radius</string>
           </property>
           <property name="font">
            <font>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Wattage</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>State</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Intensity</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Cl. Voltage</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>SVG</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGraphicsView" name="svgView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="newElement">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>New</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="editElement">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectAll">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Select All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeSelection">
          <property name="text">
           <string>Remove Selection</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="deleteLast">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Delete Last</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteSelected">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Delete Selected</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteAll">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Delete All</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>577</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Export...</string>
     </property>
     <addaction name="menuExportSVG"/>
     <addaction name="menuExportJPG"/>
     <addaction name="menuExportPDF"/>
    </widget>
    <addaction name="actionNewFile"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionSaveFile"/>
    <addaction name="menuExport"/>
   </widget>
   <widget class="QMenu" name="menuElement">
    <property name="title">
     <string>Element</string>
    </property>
    <widget class="QMenu" name="menuDelete">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Delete...</string>
     </property>
     <addaction name="menuDeleteSelected"/>
     <addaction name="menuDeleteLast"/>
     <addaction name="menuDeleteAll"/>
    </widget>
    <addaction name="actionNewElement"/>
    <addaction name="actionEditElement"/>
    <addaction name="menuDelete"/>
   </widget>
   <widget class="QMenu" name="menuBind">
    <property name="title">
     <string>Connect</string>
    </property>
    <addaction name="actionConnecAll"/>
    <addaction name="actionConnect2"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <widget class="QMenu" name="menuCreatorsOlek">
     <property name="title">
      <string>Project Creators</string>
     </property>
     <addaction name="actionAleksander_Grzybowski"/>
     <addaction name="menuCreatorsDaniel"/>
    </widget>
    <addaction name="actionApplication"/>
    <addaction name="menuCreatorsOlek"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuElement"/>
   <addaction name="menuBind"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionNewElement">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionEditElement">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionConnecAll">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>All</string>
   </property>
  </action>
  <action name="actionConnect2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Two Elements</string>
   </property>
  </action>
  <action name="actionApplication">
   <property name="text">
    <string>Application</string>
   </property>
  </action>
  <action name="actionAleksander_Grzybowski">
   <property name="text">
    <string>Aleksander Grzybowski</string>
   </property>
  </action>
  <action name="menuCreatorsDaniel">
   <property name="text">
    <string>Piotr Daniel</string>
   </property>
  </action>
  <action name="menuExportSVG">
   <property name="text">
    <string>SVG</string>
   </property>
  </action>
  <action name="menuExportJPG">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>JPG</string>
   </property>
  </action>
  <action name="menuExportPDF">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PDF</string>
   </property>
  </action>
  <action name="menuDeleteSelected">
   <property name="text">
    <string>Selected</string>
   </property>
  </action>
  <action name="menuDeleteLast">
   <property name="text">
    <string>Last</string>
   </property>
  </action>
  <action name="menuDeleteAll">
   <property name="text">
    <string>All</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
